blockreassurance.php

```diff
@@ -35,25 +35,34 @@
 
 class Blockreassurance extends Module implements WidgetInterface
 {
+    /**
+     * @var string
+     */
     private $templateFile;
 
     public function __construct()
     {
         $this->name = 'blockreassurance';
+        $this->tab = 'front_office_features';
         $this->author = 'PrestaShop';
         $this->version = '3.0.1';
 
         $this->bootstrap = true;
         parent::__construct();
 
-        $this->displayName = $this->trans('Customer reassurance', array(), 'Modules.Blockreassurance.Admin');
-        $this->description = $this->trans('Adds an information block aimed at offering helpful information to reassure customers that your store is trustworthy.', array(), 'Modules.Blockreassurance.Admin');
+        $displayName = 'Customer reassurance block';
```

Comment:
Modifying the `displayName` property requires an update of translations as well.